pyvista/plotting/tools.py

```diff
@@ -445,6 +445,64 @@ def create_axes_orientation_box(
     return actor
 
 
+def create_north_arrow():
```

Comment:
This may be better as an example mesh rather than a tool here

Reply:
Personally, this looks tool.